recipes/uhd/0001-Fix-snprintf-redeclared-as_snprintf.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/host/CMakeLists.txt b/host/CMakeLists.txt
+index 916138ee9..0bce6c726 100644
+--- a/host/CMakeLists.txt
++++ b/host/CMakeLists.txt
+@@ -260,6 +260,7 @@ if(MSVC)
+         -D_CRT_NONSTDC_NO_WARNINGS
+         -D_CRT_NONSTDC_NO_DEPRECATE
+         -D_WINSOCK_DEPRECATED_NO_WARNINGS
++        -DHAVE_SNPRINTF
```

Comment:
You can do `set "CPPFLAGS=%CPPFLAGS% -DHAVE_SNPRINTF" in bld.bat and avoid this patch.

Reply:
Don't know why I didn't try that in the first place, but yeah that's much easier.

Reply:
Looks like CMake is not picking up ``-DHAVE_SNPRINTF`` from the ``CPPFLAGS`` variable. Any pointers @isuruf?

Reply:
Looks like you need `CXXFLAGS` and `CFLAGS` as CMake ignores `CPPFLAGS`